### PR TITLE
layout desktop and mobile implemented + calculating total order price…

### PR DESCRIPTION
**Pull request by hamandamonteiro**
… with the info of the product clicked in productpage

### CODE REVIEW

File: src/components/ProductPage.js [hunk 1]
```diff
@@ -48,13 +41,20 @@ export default function ProductPage() {
...
+        if(cart.length === 0) {            
+            const newCart = [{ id: parseInt(params.id), quantity: counter, price: product.price, name: product.name}]
+            setCart(newCart);
+        } else {
+            const newCart = [...cart, { id: parseInt(params.id), quantity: counter, price: product.price, name: product.name}]
+            setCart(newCart);
+        }       
     }
 
+    console.log(cart);
```
Comment (yungtay) [inline src/components/ProductPage.js]: Faltou retirar o console.log

**Review by yungtay — CHANGES_REQUESTED** [review]
Faltou tirar o console.log

File: src/components/ProductPage.js (hunk 1 above)
Comment (hamandamonteiro) [reply src/components/ProductPage.js]: tirei hihihi valeu

Comment (yungtay) [reply src/components/ProductPage.js]: Boooa
Como aceita agora ? kk

**Review by yungtay — APPROVED** [review]

File: src/components/ProductPage.js (hunk 1 above)
Comment (yungtay) [reply src/components/ProductPage.js]: Achei kk

**Pull request merged**